src/main/java/Duke.java

```diff
@@ -78,9 +79,9 @@ private static void printError(String errorMessage) {
         System.out.println(LINE);
     }
 
-    private static void addTodo(String commandParameter) throws Exception{
+    private static void addTodo(String commandParameter) throws DukeException{
         if (commandParameter.isEmpty()){
-            throw new Exception("Task description is empty!");
+            throw new DukeException("The description of a todo task cannot be empty.");
         }
```

Comment:
I think the exception msg in this class can be put into another Exception class. Which can make the main class more clear and can add more exception msg easier.